source/form.html

```diff
@@ -0,0 +1,267 @@
+            fill="black" />
+          <path
+            d="M26.24 18.92C24.4445 20.5437 22.7512 22.277 21.17 24.11C19.5986 25.9361 17.8238 27.5768 15.88 29C15.66 28.47 14.26 27.49 14.26 27.49C14.26 27.49 11.13 25.35 9.08 23.67C7.97414 22.7373 6.9287 21.7353 5.95 20.67L3.95 20.56C6.84018 24.7655 11.0702 27.8676 15.95 29.36C16.81 29.22 19.22 27.5 21.02 26.1C23.1389 24.2819 25.124 22.3134 26.96 20.21C27.4625 19.621 27.8493 18.9425 28.1 18.21C27 18.6 26.24 18.92 26.24 18.92Z"
+            fill="#F06351" />
+        </g>
+        <defs>
+          <clipPath id="clip0_153_267">
+            <rect width="33.32" height="37.61" fill="white" />
+          </clipPath>
+        </defs>
+      </svg>
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button">
+        <span class="visually-hidden">Открыть меню</span>
+      </button>
+      <div class="main-nav__wrapper">
+        <ul class="main-nav__list">
+          <li>
+            <a class="nav__link" href="index.html">Главная</a>
+          </li>
+          <li>
+            <a class="nav__link" href="catalog.html">Каталог продукции</a>
+          </li>
+          <li>
+            <a class="nav__link" href="№">Подбор программы</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <main class="form">
+    <h1 class="form__title">Подбор программы</h1>
+    <p class="form__recommendation">Заполните анкету, и мы подберем программу питания для вашего кота</p>
+    <form class="form__selection selection" action="https://echo.htmlacademy.ru/" method="post">
+      <div class="fieldset-wrapper">
+        <fieldset class="selection__fieldset-name">
+          <ul class="selection__list">
+            <li class="selection__item">
+              <label class="selection__label">имя:*
+                <input class="selection__input" type="text" placeholder="барсик" name="name"/>
+              </label>
+            </li>
+            <li class="selection__item">
+              <label class="selection__label">Вес (кг):*
+                <input class="selection__input" type="number" placeholder="7" name="weight"/>
+              </label>
```

Comment:
Просто рекомендация: если в теге есть другой тег, то читаемее будет, если писать все содержимое на новых строках:
```suggestion
              <label class="selection__label">
                Вес (кг):*
                <input class="selection__input" type="number" placeholder="7" name="weight"/>
              </label>
```